combine.go

```diff
@@ -60,6 +65,14 @@ func (t timeList) Less(i, j int) bool { return t[i].Before(t[j]) }
 func (t timeList) Swap(i, j int)      { t[i], t[j] = t[j], t[i] }
 
 func (n *CombineNode) runCombine([]byte) error {
+	valueF := func() int64 {
+		n.cardinalityMu.RLock()
```

Comment:
Should we name this expressionsByGroupMu? I have typically named mutexes using the convention <name of thing being locked>Mu so that it is obvious, or just `mu` if it is global to the entire struct.

Thoughts? Maybe we leave the name alone but above in the struct place the cardinalityMu next to the things being locked so its clear.